packages/fx-core/src/question/scaffold/vsc/daProjectTypeNode.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Inputs, IQTreeNode } from "@microsoft/teamsfx-api";
+import { featureFlagManager, FeatureFlags } from "../../../common/featureFlags";
+import { getLocalizedString } from "../../../common/localizeUtils";
+import { QuestionNames } from "../../constants";
+import { pluginApiSpecQuestion, pluginManifestQuestion } from "../../create";
+import {
+  ApiAuthOptions,
+  ApiPluginStartOptions,
+  DACapabilityOptions,
+  setTemplateName,
+} from "./CapabilityOptions";
+import { ProjectTypeOptions } from "./ProjectTypeOptions";
+import { apiSpecNode } from "./m365ProjectTypeNode";
+
+export function daProjectTypeNode(
```

Comment:
Capability is missing in this node which will lead to project scaffolding failed for basic GPT without action. 

Previously: https://github.com/OfficeDev/teams-toolkit/blob/06b9cf03cd10b447f4dbf203a66f9c6627280aaf/packages/fx-core/src/question/create.ts#L234C14-L235C43